src/ray/gcs/client.cc

```diff
@@ -2,8 +2,8 @@
 
 #include "ray/gcs/redis_context.h"
 
-static void GetRedisShards(redisContext *context, std::vector<std::string> *addresses,
-                           std::vector<int> *ports) {
+static void GetRedisShards(redisContext *context, std::vector<std::string>& addresses,
+                           std::vector<int>& ports) {
```

Comment:
Sorry for the nit, but to keep the style consistent, we usually put the `&` with the variable name, so `&addresses` and `&ports` instead of `& addresses` and `& ports`.